project-koku/koku
# GCP customer filtered data flow

## Jira Ticket

[COST-3509](https://issues.redhat.com/browse/COST-3509)

## Description

This change will enable GCP storage only sources with post report ingestion

## Testing

1. Checkout Branch
2. Restart Koku
3. Create a storage-only GCP source - data-source includes
    {
    "bucket":"my-bucket",
    "storage-only": "True"
    },
4.Post to the new endpoint a marketplace report stored in S3
    a. Endpoint: /api/cost-management/v1/ingress/reports/
    b. Example: {"source": "9e7159f1-c5fe-4455-9881-ee730654cf84", "reports_list": ["MY_container/2023/01/18/34472771-acab-4ca4-8c6a-8721d9b91188/cc3ecf82-9960-4f02-9a31-b9163f75e6b7.csv"]}
5. See that the data is ingested correctly

## Notes

...
